Misc/NEWS

```diff
@@ -909,6 +911,8 @@ Windows
 Documentation
 -------------
 
+- bpo-28929: Link the documentation to its source file on GitHub
```

Comment:
This is unnecessary as the same bit of information is picked up in the 3.7 release notes.